www/testcancelled.inc

```diff
@@ -8,7 +8,7 @@
             $page_description = "Website speed test cancelled.";
             include ('head.inc');
         ?>
-        <link rel="stylesheet" href="<?= "/css/button.css?v={$VER_CSS}" ?>" />
+        <link rel="stylesheet" href="<?= "/css/button.css?v={$VER_BUTTON_CSS}" ?>" />
```

Comment:
@jefflembeck Does this need to be wrapped in `constant()`?

Reply:
omg I missed that entirely

Reply:
```suggestion
        <link rel="stylesheet" href="/css/button.css?v=<?= constant('VER_BUTTON_CSS') ?>" />
```